client_api.go

```diff
@@ -413,7 +438,7 @@ func (c *Client) ClaimDNS(ctx context.Context, id, authDomain string) (bool, err
 	return c.claimAssert(ctx, body, id, pathDNS)
 }
 
-// ClaimHTTP requests assertion of domain control using HTTP once the appropriate
+// ClaimHTTP requests assEmailertion of domain control using HTTP once the appropriate
```

Comment:
why was this changed ?

Reply:
My mistake that I didn't notice, thanks!